packages/web-app-files/src/components/Upload/DetailsWidget.vue

```diff
@@ -5,8 +5,15 @@
         <oc-icon name="file_copy" class="oc-mr-s" />
         <div class="uk-width-expand">
           <div class="uk-flex">
-            <div class="oc-text-bold uk-width-expand uk-text-truncate">{{ item.name }}</div>
-            <div class="uk-width-auto uk-text-nowrap">{{ getResourceSize(item.size) }}</div>
+            <div
+              id="files-upload-details-item-name"
```

Comment:
Oh sorry, seeing this now, can you please move this to either `data-testid` or to `class`? This will never be unique, as it's in a for loop. Uniqueness is required for html ids.